src/WebJobs.Script/Workers/Rpc/RpcWorkerChannelState.cs

```diff
@@ -11,21 +11,21 @@ public enum RpcWorkerChannelState
         /// <summary>
         /// The Default state of LanguageWorkerChannel.
         /// </summary>
-        Default = 0,
+        Default = 1 << 0,
 
         /// <summary>
         /// LanguageWorkerChannel is created. InvocationBuffers per function are setup
         /// </summary>
-        InvocationBuffersInitialized = 1,
+        InvocationBuffersInitialized = 1 << 1,
```

Comment:
Any reason why this change is needed?

Reply:
I was changing the Default's value ( 0 cannot be a flag value) and also updated the rest just for readability sake.

---
In reply to: [397946568](https://github.com/Azure/azure-functions-host/pull/5794#discussion_r397946568) [](ancestors = 397946568)

Reply:
instead of using bitwise operator, for simplicity can you change the values to : 1, 2, 4 ....?

Reply:
I kinda prefer the bitwise operator too - it makes it easier for then next person to code and not make a math mistake!

Reply:
alritie..which ever makes it easier to extend this. Will resolve this comment.